example/README.md

```diff
@@ -0,0 +1,9 @@
+# Run the Example App
```

Comment:
All info is included here: https://github.com/callstack/react-native-pager-view/blob/master/CONTRIBUTING.md

Reply:
Thank you @troZee.

Maybe a link to this file from the above README.md might be a good idea then, instead? It would not have occurred to me to look into CONTRIBUTING.md, as I was just trying to see the PagerView in action...